charts/prometheus-nginx-exporter/templates/deployment.yaml

```diff
@@ -0,0 +1,76 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "prometheus-nginx-exporter.fullname" . }}
+  namespace: {{ template "prometheus-nginx-exporter.namespace" . }}
+  {{- with .Values.additionalAnnotations }}
+  annotations:
+  {{ toYaml . | indent 4 }}
+  {{- end }}
+  labels:
+    {{- include "prometheus-nginx-exporter.labels" . | indent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "prometheus-nginx-exporter.selectorLabels" . | indent 6 }}
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        {{- include "prometheus-nginx-exporter.labels" . | indent 8 }}
+      annotations:
+      {{- with .Values.additionalAnnotations }}
+      {{ toYaml . | indent 6 }}
+      {{- end }}
+    spec:
+      serviceAccountName: {{ template "prometheus-nginx-exporter.serviceAccountName" . }}
+      {{- with .Values.initContainers }}
+      initContainers: {{ toYaml . | nindent 8 }}
+      {{- end }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
```

Comment:
would be nice the default image tag could be used from the Chart.yaml appversion value.

Example: https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-conntrack-stats-exporter/templates/daemonset.yaml#L35